<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Putty++ Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="widget_1" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QWidget" name="LoginParms" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>420</width>
        <height>220</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>420</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>420</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#LoginParms{
	border:1px solid #dcdcdc
}</string>
      </property>
      <widget class="QLineEdit" name="editHostName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>280</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Session Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="editPassword">
       <property name="geometry">
        <rect>
         <x>258</x>
         <y>140</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>150</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="editUserName">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>140</y>
         <width>231</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBoxPro" name="editPort">
       <property name="geometry">
        <rect>
         <x>308</x>
         <y>90</y>
         <width>100</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>210</width>
         <height>30</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_0">
        <widget class="QPushButton" name="pushButtonEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonRemove">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_1">
        <widget class="QPushButton" name="pushButtonSave">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Conn Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="connType">
       <property name="geometry">
        <rect>
         <x>308</x>
         <y>40</y>
         <width>100</width>
         <height>25</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedWidgetAddress">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>74</y>
         <width>285</width>
         <height>50</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_ip">
        <widget class="QLineEdit" name="editAddress">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>280</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>0</y>
           <width>230</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Host Name (or IP Address)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_serial">
        <widget class="QLineEdit" name="editSerial">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>280</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>COM1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>0</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Serial line</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
      <zorder>stackedWidgetAddress</zorder>
      <zorder>stackedButton</zorder>
      <zorder>editHostName</zorder>
      <zorder>label_2</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>editPassword</zorder>
      <zorder>label_6</zorder>
      <zorder>editUserName</zorder>
      <zorder>editPort</zorder>
      <zorder>label_7</zorder>
      <zorder>connType</zorder>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>0</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Session</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>30</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButtonClose">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLogIn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSet">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(://res/set.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidgetPro" name="LoginSite">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#LoginSite::item {
	padding: 6px;
	border: 2px solid transparent;
	border-radius: 6px;
}
QListWidget#LoginSite::item:selected {
	background: #E3F2FD;
	border-color: #0078d4;
	color: #000000
}
QListWidget#LoginSite::item:hover {
	background: #E3F2FD;
}
</string>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QListWidgetPro</class>
   <extends>QListWidget</extends>
   <header>qlistwidgetpro.h</header>
  </customwidget>
  <customwidget>
   <class>QSpinBoxPro</class>
   <extends>QSpinBox</extends>
   <header>qspinboxpro.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
